docs/serving/_index.md

```diff
@@ -22,22 +22,22 @@ Knative Serving defines a set of objects as Kubernetes Custom Resource
 Definitions (CRDs). These objects are used to define and control how your
 serverless workload behaves on the cluster:
 
-- [Service](https://github.com/knative/docs/blob/main/docs/serving/spec/knative-api-specification-1.0.md#service):
+- [Service](https://github.com/knative/docs/blob/main/docs/serving/spec/knative-api-specification-1.0#service):
```

Comment:
Not for this PR, but I wonder whether linking to the spec is the best link here.

Reply:
Oh, but for this PR, it turns out that these links are probably dead and also not relative (relative links allow the version to be preserved).